index.html

```diff
@@ -0,0 +1,723 @@
+          a trillion trillionfold — from a submicroscopic speck of primordial
+          energy to the size of a grapefruit — in less than a cosmic eye-blink.
+        </p>
+        <p>
+          Almost as quickly, this pulse would subside, relaxing into ordinary
+          particles and radiation. All of normal cosmic history was still ahead,
+          resulting in today’s observable universe, a patch of sky and stars
+          billions of light-years across. “It’s often said that there is no such
+          thing as a free lunch,” Dr. Guth likes to say, “but the universe might
+          be the ultimate free lunch.”
+        </p>
+        <p>
+          Make that free lunches. Most of the hundred or so models resulting
+          from Dr. Guth’s original vision suggest that inflation, once started,
+          is eternal. Even as our own universe settled down to a comfortable
+          homey expansion, the rest of the cosmos will continue blowing up,
+          spinning off other bubbles endlessly, a concept known as the
+          multiverse.
+        </p>
+        <p>
+          So the future of the cosmos is perhaps bright and fecund, but do not
+          bother asking about going any deeper into the past.
+        </p>
+        <p>
+          We might never know what happened before inflation, at the very
+          beginning, because inflation erases everything that came before it.
+          All the chaos and randomness of the primordial moment are swept away,
+          forever out of our view.
+        </p>
+        <p>
+          “If you trace your cosmic roots,” said Abraham Loeb, a
+          Harvard-Smithsonian astronomer who was not part of the team, “you wind
+          up at inflation.”
+        </p>
+      </div>
+
+      <!--SUGGESTED BOTTOM SECTION two columns with one big column with 3 columns inside it-->
+      <section class="suggestednewsletters">
+        <div class="column-footer-two">
+          <h3>Most Popular</h3>
+          <ul class="no-bullets">
+            <li>
+              Trump, in Scorching Attack on McConnel,Urges G.O.P. to Replace Him
+            </li>
+            <li>Is This the Most Radical Film Ever Produced by Hollywood?</li>
+            <li>
+              Stephen Colbert Gets Serious About New Insurrection Evidence
+            </li>
+            <li>Jimmy Kimmel Knows How the Impeachment Sequel Will End</li>
+            <li>Was '60 Minutes' TV's Most Toxic Workplace?</li>
+            <li>
+              Trump Tower's Neighbors Are Relieved as Barricades Come Down
+            </li>
+            <li>
+              I'm Afraid He's Going to Run Again and Lose; Lieu Says of Trump
+            </li>
+            <li>
+              Walking (and Skating) on Thin Ice, Some in Europe Paid the Price
+            </li>
+            <li>
+              The Deaths of teenage Cousins in a Village in India Have Global
+              Ramifications
```

Comment:
## Improve this section
- [ ] Images in this section are not loading
- [ ] The elements are widely spaced. they should not be covering the entire page.
- [ ] Implement the missing `Space and Astronomy` section as well.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/108400184-fdabff80-722b-11eb-9511-f199db7fd1bf.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/108400224-056ba400-722c-11eb-9925-fcf2208554e9.png)|

Reply:
The images and the paragraph below are not well aligned